src/api-doc/api-doc.yaml

```diff
@@ -2230,3 +2230,634 @@ paths:
+                              type: string
+                            entityTypeId:
+                              type: string
+                      updatedAt:
+                        type: string
+                      createdAt:
+                        type: string
+                      __v:
+                        type: number
+              examples:
+                SuccessResponse:
+                  value:
+                    message: USER_ROLE_INFORMATION_CREATED
+                    status: 200
+                    result:
+                      status: ACTIVE
+                      createdBy: SYSTEM
+                      updatedBy: SYSTEM
+                      _id: 66a9da8ab0f18a5b3f1c824f
+                      deleted: false
+                      userRoleId: 13
+                      title: DC
+                      userType: 1
+                      entityTypes:
+                        - _id: 66a9da8ab0f18a5b3f1c8250
+                          entityType: district
+                          entityTypeId: 5f32d8228e0dc8310056e
+                      updatedAt: '2024-07-31T06:32:42.097Z'
+                      createdAt: '2024-07-31T06:32:42.097Z'
+                      __v: 0
+        '400':
+          description: Bad Request.
+          headers: &ref_13 {}
+          content:
+            application/json:
+              schema:
+                $schema: http://json-schema.org/draft-04/schema#
+                type: object
+                properties:
+                  status:
+                    type: number
+                  message:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        location:
+                          type: string
+                        param:
+                          type: string
+                        msg:
+                          type: string
+                      required:
+                        - location
+                        - param
+                        - msg
+              examples:
+                FailureResponse:
+                  value:
+                    status: 400
+                    message:
+                      - location: body
+                        param: title
+                        msg: 'required title '
+                      - location: body
+                        param: title
+                        msg: 'required title '
+                      - location: body
+                        param: entityTypes[0].entityType
+                        msg: 'entityType name '
+                      - location: body
+                        param: entityTypes[0].entityTypeId
+                        msg: 'entityTypeId name '
+  /v1/userRoleExtension/update/66a94c06fff8abb34f9d800f:
+    post:
+      summary: To update a single user-role data
+      description: >-
+        You can use this API to to update user-role data.
```

Comment:
@MallanagoudaB not user role it is user role extension. please check entire file for similar chages